charts/oes/charts/spinnaker/templates/configmap/halyard-init-script.yaml

```diff
@@ -68,10 +68,12 @@ data:
     if [ -f /tmp/spinnaker/.hal/default/profiles/fiat-local.yml ]; then
     sed -i  s/RELEASE_NAME/{{ .Release.Name }}/g /tmp/spinnaker/.hal/default/profiles/fiat-local.yml
     fi
+    sed -i  s/SPINNAKER_NAMESPACE/${SPINNAKER_NAMESPACE}/ /tmp/spinnaker/.hal/default/profiles/orca-local.yml
     printf 'server.address: 0.0.0.0\n' > /tmp/config/halyard-local.yml
     if [ -f /tmp/spinnaker/.hal/halyard.yaml ]; then
     cp /tmp/spinnaker/.hal/halyard.yaml /tmp/config
     fi
+    chmod 777 -R /tmp/spinnaker/.hal
```

Comment:
What permissions do the files have after they are copied to /home/spinnaker/.hal directory. I think giving 777 would violate least privilege principle.

Reply:
Leave it for now for openshift testing. Will remove later if required.